modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -25,16 +25,32 @@ For more information about how to install Helm, see the Helm installation docume
 * Not installed Istio in the cluster
 
 If you want to enable automatic sidecar injection, you must additionally have:
+
 * Install Kubernetes 1.12+ with admissionregistration API 
++
 *Tip:* To verify the installation, use the command `kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
-* Configure kube-apiserver process with the admission-control flag to “MutatingAdmissionWebhook”
-For more information about how to turn on, see tke Kubernetes documentation.
+* Configure kube-apiserver process with the admission-control flag to “MutatingAdmissionWebhook”.
++
 * Add and list ValidatingAdmissionWebhook admission controllers in the correct order.
 
+== Hardware Requirements
+
+Your Anypoint Service Mesh installation requires the following hardware configuration:
+
+* CPU: 4 cores
+* Memory: 8 GBs
+
 
 == Licensing Requirements
 
-You require a Anypoint Service Mesh license to perform the installation. If you do not have a license, contact your MuleSoft representative.
+You require a Anypoint Service Mesh license to perform the installation. Ensure that the license you specify is in a digested format. If you do not have a license, contact your MuleSoft representative.
+
+Note that the license you procure must contain the following entitlements:
+
+* API Gateway: api-gateway
+* Cluster: clustering
+
+*Need to add more infomation about the digested format.--Tomas*
```

Comment:
For the time being, we will leave it as is.

Reply:
Done! Thanks!